src/OpaEarning/Build/Deploy/1718/sql/dml/01 Ilr.Transient.OPA.Populate.Reference.dml.sql

```diff
@@ -243,7 +244,7 @@ INSERT INTO [Reference].[AEC_LatestInYearEarningHistory] (
         [UKPRN],
         [ULN],
 		[LatestInYear]
-    FROM ${ILR_Deds.FQ}.[Version_001].[AEC_LatestInYearEarningHistory]
+    FROM ${DS_SILR1718_Collection.databasename}.[Version_001].[AEC_LatestInYearEarningHistory]') as oq
     WHERE [ULN] IN (SELECT [ULN] FROM [Valid].[Learner])
```

Comment:
This is outside the scope of this story, but would an inner join on Valid.Learner improve the performance?